src/Identifier/Identifier.php

```diff
@@ -27,6 +38,19 @@ public function matches(string $pattern, string $value, array $excludes = []): b
 				}
 			}
 		}
+		if ($matches && $excludeWithAttributes) {
+			$attributes = class_exists($value) || interface_exists($value)
+				? array_map(fn ($a) => $a->getName(), $this->reflector->reflectClass($value)->getAttributes())
```

Comment:
```suggestion
				? array_map(fn($a) => $a->getName(), $this->reflector->reflectClass($value)->getAttributes())
```

I'm aware this is not covered by the coding standard check, but let's use what is used elsewhere in the lib.